facia/app/controllers/FaciaController.scala

```diff
@@ -200,7 +200,8 @@ trait FaciaController
       case Some((faciaPage, _)) if nonHtmlEmail(request) =>
         successful(Cached(CacheTime.RecentlyUpdated)(renderEmail(faciaPage)))
       case Some((faciaPage: PressedPage, targetedTerritories))
-          if FaciaPicker.getTier(faciaPage, path)(request) == RemoteRender =>
+          if FaciaPicker.getTier(faciaPage, path)(request) == RemoteRender
+            && !request.isJson =>
```

Comment:
Tempted to refactor RemoteRenderingService so that it doesn't need to return a `Cached` type and we don't need to handle this seperately, but that seems like a LOT of work.